soc/gd_gd32/gd32vf103/Kconfig

```diff
@@ -4,17 +4,17 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_GD32VF103
-	bool "GigaDevice GD32VF103 series SoC implementation"
 	select RISCV
 	select RISCV_PRIVILEGED
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
+	select RISCV_ISA_EXT_C
+	select RISCV_ISA_EXT_ZICSR
+	select RISCV_ISA_EXT_ZIFENCEI
+	select RISCV_HAS_CLIC
 	select ATOMIC_OPERATIONS_C
 	select INCLUDE_RESET_VECTOR
-	select BUILD_OUTPUT_HEX
-	select XIP
 	select GD32_HAS_AFIO_PINMUX
 	select GD32_HAS_IRC_40K
-	select HAS_GD32_HAL
```

Comment:
shouldn't `HAS_GD32_HAL` still be selected?

Reply:
it's done by

```
config SOC_FAMILY_GD_GD32
	select HAS_GD32_HAL
	select BUILD_OUTPUT_HEX
	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
```